lib/Extension/LanguageServerPhpstan/Tests/LanguageServerPhpstanExtensionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpstan\Tests;
+
+use Phpactor\Container\Container;
+use Phpactor\Container\PhpactorContainer;
+use Phpactor\Extension\LanguageServerPhpstan\Model\Linter;
+use Phpactor\Extension\LanguageServerPhpstan\Model\Linter\PhpstanLinter;
+use Phpactor\Extension\Logger\LoggingExtension;
+use Phpactor\Extension\FilePathResolver\FilePathResolverExtension;
+use Phpactor\Extension\LanguageServerPhpstan\LanguageServerPhpstanExtension;
+use PHPUnit\Framework\TestCase;
+
+class LanguageServerPhpstanExtensionTest extends TestCase
```

Comment:
@dantleech I added this test class to confirm that params are properly handled: https://github.com/phpactor/phpactor/pull/2764/commits/45422cb944ff834d87ef4be23d27b257bc9c019b

I am currently only testing the `tmp_file_disabled` param, but it could be extended to others.

_Caveat: I added a public `isTmpFileDisabled` to the `PhpstanLinter` for this._

I'm not sure if this aligns with your test setup, so feel free to remove it again.